quil-rs/src/program/mod.rs

```diff
@@ -702,7 +702,7 @@ mod tests {
             Gate, Instruction, Jump, JumpUnless, JumpWhen, Label, Matrix, MemoryReference, Qubit,
             QubitPlaceholder, Target, TargetPlaceholder,
         },
-        quil::Quil,
+        quil::{Quil, INDENTATION as INDENT},
```

Comment:
Why the rename? Why not just call it `INDENT` everywhere?

Reply:
Fair, I did that last minute while trying to fix the test. I renamed the constant to INDENT.